dev/MenuBar/MenuBarItem.xaml

```diff
@@ -67,12 +71,38 @@
 
                         <Button x:Name ="ContentButton"
                             Content="{TemplateBinding Title}"
-                            Background="Transparent"
-                            BorderThickness="0"
-                            VerticalAlignment="Stretch"
-                            Padding="12,0,12,0"
-                            IsTabStop="False"
-                            AutomationProperties.AccessibilityView="Raw"/>
+                            IsTabStop="False">
+                            <Button.Style>
```

Comment:
It's my first time re-templating, so if someone knows a better way to make it so the button foreground brush stays consistent through all visual states except disabled, let me know!

Reply:
You can either put the button style in ContentRoot Grid resources or just create a style for this button in this file and then reference it using Static/ThemeResource :)